Update event.adoc (#172)
starknet-edu/starknetbook: chapters/book/modules/chapter_2/pages/event.adoc

== Defining Events

> To define an event in a Starknet contract, you can use the **`#[event]`** attribute followed by defining an **`enum`** that will contain the event variants. 

Reviewer: ```suggestion
To define an event in a Starknet contract, you can use the **`#[event]`** attribute followed by defining an **`enum`** that contains the event variants. 
```

Reviewer: @OkoliEvans I don't see that my comments were implemented. 

Just to be clear, the changes I requested are based on our style guide and general writing guidelines.

For more information, see the [Starknet documentation supplementary style guide](https://github.com/starknet-io/starknet-docs/blob/dev/contributing_to_docs/starknet_docs_style_guide.adoc). and [Contributing to Starknet documentation](https://github.com/starknet-io/starknet-docs#contributing-to-starknet-documentation).

[source,rust]
----
#[event]
#[derive(Drop, starknet::Event]
enum Event {
    VoteCast: VoteCast,
    UnauthorizedAttempt: UnauthorizedAttempt,
}
----

> In our voting contract, we have two events, **`VoteCast`** and **`UnauthorizedAttempt`** which are defined in the enum above. The **`VoteCast`** event is emitted when a vote is cast. The event takes two parameters: the voter's address and the vote value (0 for No, 1 for Yes).

Reviewer: ```suggestion
The voting contract has two events, *`VoteCast`* and *`UnauthorizedAttempt`*, which are defined in the enum above. The *`VoteCast`* event is emitted when a vote is cast. The event takes two parameters: the voter's address and the vote value: 0 for No, 1 for Yes.
```
